mlflow/server/js/src/common/utils/StringUtils.js

```diff
@@ -13,3 +13,17 @@ export const capitalizeFirstChar = (str) => {
   }
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
 };
+
+export const middleTruncateKey = (str, maxLen) => {
```

Comment:
```suggestion
export const middleTruncate = (str, maxLen) => {
```

Reply:
or do you prefer `middleTruncateKey`?

Reply:
middleTruncateStr looks better

Reply:
`middleTruncateStr` sounds good to me :)